### ISSUE TITLE
Wrong concat of quoted strings

### ISSUE THREAD

**Issue by gilles-duboscq** [chimpler/pyhocon#84]
Pyhocon ignores white spaces between quoted strings during string concatenation.

Given the following config:
```hocon
a = "foo" "bar"
```
The Java implementation gives an object that, in JSON looks like:
```json
{
    "a" : "foo bar"
}
```
while pyhocon gives:
```json
{
    "a" : "foobar"
}
```

**Comment by darthbear** [comment]
Thanks @gilles-duboscq  for the bug report! It's fixed in `pyhocon==0.3.28`

**Comment by gilles-duboscq** [comment]
Great, thanks!
